internal/server/server.go

```diff
@@ -351,6 +352,33 @@ func runScheduleRuleChecker(exit <-chan struct{}) {
 	runScheduleRuleCheckerByInterval(d, exit)
 }
 
+func handleAllRuleStatusMetrics(rs []ruleWrapper) {
+	if conf.Config != nil && conf.Config.Basic.Prometheus {
```

Comment:
This seems like setting in initialization. Do the metrics updated in runtime when start/stop rules?

Reply:
The metircs updated in every interval (10s)